### PR TITLE
due date time needs start date time

### PR DESCRIPTION
**Pull request by verhyppo**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14820b3](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/14820b3365336a7368e337cbf38783f53f56a1e0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14820b3](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/14820b3365336a7368e337cbf38783f53f56a1e0)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[14820b3](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/14820b3365336a7368e337cbf38783f53f56a1e0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/followupflag.md](https://github.com/verhyppo/microsoft-graph-docs/blob/patch-2/api-reference/v1.0/resources/followupflag.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251350103522-4043%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o3%2BwiiRm1obRsdRF8tlemhNe5a6jN4y5bCZwjJQw%2Bs8%3D&st=2019-02-25T13%3A48%3A47Z&se=2019-03-28T13%3A53%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6730ecb](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/6730ecb22bd65ac6e52a0181b6cdcf37cb013b47)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6730ecb](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/6730ecb22bd65ac6e52a0181b6cdcf37cb013b47)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6730ecb](https://github.com/verhyppo/microsoft-graph-docs/pull/4043/commits/6730ecb22bd65ac6e52a0181b6cdcf37cb013b47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/followupflag.md](https://github.com/verhyppo/microsoft-graph-docs/blob/patch-2/api-reference/v1.0/resources/followupflag.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251722084476-4043%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=M1kc0sSK8xYUm9S%2Bkpxh3eaJPzh3jcVa3O40KwwxNLw%3D&st=2019-02-25T17%3A20%3A41Z&se=2019-03-28T17%3A25%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/v1.0/resources/followupflag.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ Allows setting a flag in an item for the user to follow up on later.
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
 |completedDateTime|[dateTimeTimeZone](datetimetimezone.md)|The date and time that the follow-up was finished.|
-|dueDateTime|**dateTimeTimeZone**|The date and time that the follow-up is to be finished.|
+|dueDateTime|**dateTimeTimeZone**|The date and time that the follow up is to be finished. **Note**: in order to set the due date, you must also specify the `startDateTime`; otherwise, you will get a `400 "InvalidRequest"` response.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/followupflag.md]: Please apply the same update to the beta version of this topic as well. Thanks.

Comment (verhyppo) [reply api-reference/v1.0/resources/followupflag.md]: Done.

**Comment by Lauragra** [comment]
@angelgolfer-ms , please review and verify this PR.

**Pull request merged**